internal/dumper/dumper.go

```diff
@@ -251,7 +257,7 @@ func (d *Dumper) dumpTable(conn *Connection, database string, table string) erro
 		where = fmt.Sprintf(" WHERE %v", v)
 	}
 
-	cursor, err := conn.StreamFetch(fmt.Sprintf("SELECT %s FROM `%s`.`%s` %s", strings.Join(selfields, ", "), database, table, where))
+	cursor, err := conn.StreamFetch(fmt.Sprintf("SELECT %s FROM `%s`.`%s` %s", strings.Join(selfields, ", "), databaseHandle, table, where))
```

Comment:
Thinking we could add test coverage to dumper_test for this one.

That file is pretty complex, I can look later myself as well.

Reply:
I went ahead and took a stab at adding test coverage, LMK what you think